{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": []
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "cve@mitre.org",
      "ID": "CVE-2021-46179"
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "Reachable Assertion vulnerability in upx before 4.0.0 allows attackers to cause a denial of service via crafted file passed to the the readx function."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": []
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "https://github.com/upx/upx/issues/545",
          "refsource": "MISC",
          "tags": [],
          "url": "https://github.com/upx/upx/issues/545"
        }
      ]
    }
  },
  "impact": {},
  "lastModifiedDate": "2023-08-22T20:10Z",
  "publishedDate": "2023-08-22T19:16Z"
}